### DISCUSSION TITLE
Class 'Spatie\GoogleCalendar\Event' not found

### DISCUSSION THREAD

**Discussion by Jhamende** [spatie/laravel-google-calendar#181] [Q&A]
Dear all, 

First of allI wich you a merry Christmas and Happy "end of Year" :) 

I followed all steps but am getting error message Class 'Spatie\GoogleCalendar\Event' not found. 
I have the folder spatie under vendor and got the google-calendar.php under config Folder and added used 
use Spatie\GoogleCalendar\Event;
at the top of my controlle. 

I tried to manually add into my config\App.php
Spatie\GoogleCalendar\GoogleCalendarServiceProvider::class,

What am I missing ? 

Thanks in advance,
Jonathan

**Comment by mikeSxz** [comment]
Hi, I experienced the same problem, I know this is 2022 and probably you already addressed that problem, but, even here in 2022 there are some newbies like me with the same problem, so, here is a URL that helped me to fix the problem, in my case I forgot to add the providers in the app.php file, and also the alias.

[https://www.larablocks.com/package/spatie/laravel-google-calendar](https://www.larablocks.com/package/spatie/laravel-google-calendar)
